packages/webapp/package.json

```diff
@@ -91,7 +91,6 @@
     "postcss-simple-vars": "^7.0.1",
     "prettier": "^2.6.2",
     "serwist": "^9.0.9",
-    "subscriptions-transport-ws": "^0.9.19",
```

Comment:
I see its a dependency of `serwist`, but I guess we don't need it in our own deps, it will just be installed from peers.